src/utils/image_server.cljs

```diff
@@ -1,4 +1,4 @@
-(ns status-im.utils.image-server
+(ns utils.image-server
   (:require [quo.design-system.colors :as colors]))
```

Comment:
please remove current-theme and quo.design-system.colors

Reply:
probably timestamp as well, it should be in utils.dates

Reply:
I'll first check status-go implementation of passing theme property to get identicon and account image uris from media server.

Reply:
I checked status-go and seems like the `theme` query url is passed for no reason : `https://github.com/status-im/status-go/blob/develop/server/handlers.go` and can be safely removed

Reply:
removing theme can impact identicon ring colour : 
https://github.com/status-im/status-go/blob/develop/server/handlers.go#L173-L185

Reply:
i didn't mean to remove it from server, i just mean from `utils.image-server` namespace, sorry for confusion

Reply:
`utils.image-server` should know nothing about `quo.design-system.colors`

Reply:
Oh i see what you mean, thanks for clarifying :)

Reply:
addressed over here : https://github.com/status-im/status-mobile/pull/15712/commits/d1eea8c121334e36dafcfae72d7ddd8f17d8e521